### PR TITLE
feat(web): album card hover styling

### PR DESCRIPTION
**Pull request by alextran1502**
Prototype - Taking inspiration from GoogleIO web page to prominently show which album is hovered
image

# Immich Implementation

image

image

### CODE REVIEW

File: web/src/lib/components/album-page/album-card.svelte [hunk 1]
```diff
@@ -61,35 +63,39 @@
...
-	<div class={`aspect-square z-[-1]`}>
+	<div class={`aspect-square relative`}>
 		<img
 			src={imageData}
 			alt={album.id}
 			class={`object-cover h-full w-full transition-all z-0 rounded-xl duration-300 hover:shadow-lg`}
 			data-testid="album-image"
 			draggable="false"
 		/>
+		<div class="w-full h-full absolute top-0 rounded-xl" bind:this={imageOverlayElement} />
```
Comment (michelheusschen) [inline web/src/lib/components/album-page/album-card.svelte:91]: There's no need to use a component binding. You can use `style:background` or preferably css classes like this:

```html
<div id="album-card" class="group ...">
	<div>
		<img />
		<div class="group-hover:bg-indigo-800/20 ..." />
	</div>
</div>
```

File: web/src/lib/components/album-page/album-card.svelte [hunk 2]
```diff
@@ -61,35 +63,39 @@
...
 	</div>
 
 	<div class="mt-4">
 		<p
-			class="text-sm font-medium text-gray-800 dark:text-immich-dark-primary"
+			class="text-2xl font-bold dark:text-immich-dark-primary text-immich-primary w-full truncate"
```
Comment (michelheusschen) [inline web/src/lib/components/album-page/album-card.svelte:96]: What do you think about `font-semibold`? Just a little too bold for my liking

File: web/src/lib/components/album-page/album-card.svelte [hunk 3]
```diff
@@ -61,35 +63,39 @@
 </script>
 
 <div
-	class="hover:cursor-pointer mt-4 relative"
+	class="hover:cursor-pointer mt-4 border-2 border-transparent dark:hover:border-gray-300 hover:border-black rounded-xl p-5"
```
Comment (michelheusschen) [inline web/src/lib/components/album-page/album-card.svelte:66]: What do you think about using `rounded-3xl`?

imageimage

File: web/src/lib/components/album-page/album-card.svelte (hunk 1 above)
Comment (alextran1502) [reply web/src/lib/components/album-page/album-card.svelte:91]: My gosh I didn't know we can refer parent element in tailwind like this! Muchhhh appreciate you for poiting out this! You are a wizard!

**Review by michelheusschen — APPROVED** [review]

**Pull request merged**